roles/kubespray-defaults/defaults/main.yaml

```diff
@@ -118,9 +118,12 @@ k8s_image_pull_policy: IfNotPresent
 efk_enabled: false
 enable_network_policy: false
 
+## When OpenStack is used, Cinder version can be explicitly specified if autodetection fails (https://github.com/kubernetes/kubernetes/issues/50461)
+openstack_blockstorage_version: "v1/v2/auto (default)"
```

Comment:
You can't set a default value that is not valid. It should be commented out or set to auto (with the comment in the preceding line)